examples/deepspeech/README.md

```diff
@@ -96,27 +97,55 @@ For example, to use GPUs 1 and 2, set CUDA_VISIBLE_DEVICES=1,2 and to use all GP
 CUDA_VISIBLE_DEVICES=0,1,2,3. The in-graph replication method for data-parallel, synchronized training implemented in
 Mozilla DeepSpeech will be used.
 
+## Using run-cluster.sh to run distributed benchmarks
+
+The Mozilla DeepSpeech implementation supports distributed training using Distributed TensorFlow with a
+parameter server and worker processes. The 'bin/run-cluster.sh' script is helpful for configuring and running
+these processes on a single machine. The script accepts an argument in a p:w:g format, where p denotes the 
+number of used parameter servers, w denotes the number of worker processes and g denotes the number of
+GPUs used by individual worker processes.
+
+For example, running our benchmark on the distributed training
+application using 1 parameter server and 4 workers using 1 GPU each would require executing the following command:
+
+```bash
+LD_LIBRARY_PATH=native_client/ bin/run-cluster.sh 1:4:1 --script="python3 DeepSpeech.py" --train_files=ldc93s1/ldc93s1.csv --dev_files=ldc93s1/ldc93s1.csv --test_files=ldc93s1/ldc93s1.csv --train_batch_size=64 --epoch=1000 --benchmark_warmup_steps=10 --benchmark_steps=10 --log_level=3
```

Comment:
+ --noshow_progressbar as it is currently not working with distributed training

Reply:
Isn't --log_level=3 enough?

Reply:
```suggestion
LD_LIBRARY_PATH=native_client/ bin/run-cluster.sh 1:4:1 --script="python3 DeepSpeech.py" --train_files=ldc93s1/ldc93s1.csv --dev_files=ldc93s1/ldc93s1.csv --test_files=ldc93s1/ldc93s1.csv --train_batch_size=64 --epoch=1000 --benchmark_warmup_steps=10 --benchmark_steps=10 --log_level=3 --noshow_progressbar
```

Reply:
> Isn't --log_level=3 enough?

unofrtunately no